src/leap/mail/mail.py

```diff
@@ -744,7 +746,7 @@ def __cb_signal_unread_to_ui(self, unseen):
         :param unseen: number of unseen messages.
         :type unseen: int
         """
-        emit_async(catalog.MAIL_UNREAD_MESSAGES, str(unseen))
+        emit_async(catalog.MAIL_UNREAD_MESSAGES, self.store.uuid, str(unseen))
```

Comment:
That will require changes in `bitmask_client/src/leap/bitmask/gui/mail_status.py` I don't think `_mail_handle_imap_events_slot` does handle the uuid. And probably we should update the comment at `leap_pycommon/src/leap/common/events/catalog.py` describing correctly the usage of this event.